.github/workflows/release-installer.yaml

```diff
@@ -0,0 +1,105 @@
+# When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# When the installers are ready in installer private bucket, run this workflow to test them.
+# TODO: Add job of uploading installers to Github release when installer tests are stable.
+# "release" runner tag is to make target hosts deterministic for easy clean up.
+# TODO: Remove the "release" runner tag when installer tests are stable.
+name: Release Installer
+on:
+  workflow_dispatch:
+
+permissions:
+  id-token: write   # This is required for requesting the JWT
+  contents: read    # This is required for actions/checkout
+
+jobs:
+  # TODO: fix the arm64 test installer.
+  # Currently the first time of calling any Finch command in arm64 hosts triggered by Github action will fail by
+  # the error "Error: Process completed with exit code 137." So the arm64 job will fail.
+  # We temporarily use follow-up manual steps to complete and clean up it.
```

Comment:
Which manual steps specifically? Should we note that somewhere?

Reply:
Just same steps after e2e tests.
```
INSTALLED=true make test-e2e
echo 'y' | sudo bash /Applications/Finch/uninstall.sh
rm -rf Finch-${GITHUB_REF_NAME}-aarch64.pkg
```
Do you think this should be added to the comment?

Reply:
My understanding (maybe incorrect):

1. `INSTALLED=true make test-e2e` step below would fail.
2. We then SSH to the host to manually run `INSTALLED=true make test-e2e`, and it should pass.
3. We then manually run the cleanup commands (i.e., `echo 'y' | sudo bash /Applications/Finch/uninstall.sh` and `rm -rf Finch-${GITHUB_REF_NAME}-aarch64.pkg`)

My question:

Instead of manual retry, Have we considered retrying the `INSTALLED=true make test-e2e` step in the workflow?

Reply:
I see. That's really weird. Since we still need to SSH to the host even if it's broken into 2 workflows, we may just do the following instead:

1. As you've already mentioned, create an issue to document and also track this.
2. At [the step that's expected to fail](https://github.com/runfinch/finch/pull/97/files#diff-4efbc8a659c1b30475339c903cf017cee4fc608ed879624d82518a24968755b3R57), link the issue and write comments to provide step-by-step instructions for folks to follow.